tests/test_ssl_transport_close.py

```diff
@@ -0,0 +1,156 @@
+import unittest
+from aiohttp import web
+import aiohttp
+from pathlib import Path
+import asyncio
+import ssl
+
+import logging
+
+import tracemalloc
+tracemalloc.start(40)
+
+
+log = logging.getLogger()
+log.setLevel(logging.DEBUG)
+logging.basicConfig(level=logging.WARNING)
```

Comment:
It's pretty uncommon to have random logs in tests. Also, such a setup is going to influence the whole runtime which is bad.